spec/features/publisher_information/visit_publisher_information_pages_spec.rb

```diff
@@ -56,6 +58,18 @@ def then_i_can_see_how_to_use_publisher_page
     end
   end
 
+  def when_i_click_on_the_request_training_link_in_footer
```

Comment:
Minor: we could DRY-up these two groups of methods.

```
def when_i_click_on_the_request_training_link_in_footer
  when_i_click_on_the_link_in_footer("Request Content Publisher training")
end
```

and similarly for the `then` steps.

Reply:
Ooh do we have much/any precedence for helper methods? I'm going to put them in but not phrase them like steps to try avoid blurring the line between step methods and helper methods.

Reply:
We used to, but I can't see any now.

Reply:
Sure, pushed it up now.

Reply:
Looks good to me 👍